django/templatetags/i18n.py

```diff
@@ -98,7 +98,8 @@ def render(self, context):
 class BlockTranslateNode(Node):
 
     def __init__(self, extra_context, singular, plural=None, countervar=None,
-                 counter=None, message_context=None, trimmed=False, asvar=None):
+                 counter=None, message_context=None, trimmed=False, asvar=None,
+                 tag_name='blocktrans'):
```

Comment:
Why are we preferring the old alias here?

Reply:
I think just a case I missed.  -- I'll update that to use the new alias.